pkg/errors/error.go

```diff
@@ -40,4 +40,6 @@ var (
 	ErrUnmarshallFailed        = errors.New("failed to unmarshall from json")
 	ErrClusterNotFound         = errors.New("Cluster instance not found in store")
 	ErrDuplicateBrickPath      = errors.New("Duplicate brick entry")
+	ErrVolNotDistribute        = errors.New("Not a distribute volume")
```

Comment:
It will be preferrable to define this withing these errors in the rebalance package itself.

Reply:
Will do.